### PR TITLE
minor: cleaning up readme and fixings vars

### PR DESCRIPTION
**Pull request by waleedakramkhan**
https://k6scope.atlassian.net/browse/KE-805

### CODE REVIEW

File: aws/README.md [hunk 1]
```diff
@@ -1,26 +1,63 @@
...
 
-The ```main.tf``` file shows how to import this module. We use Terraform's concept of local modules for now.
+- Terraform installed (version >=1.1.9)
+- AWS CLI configured with appropriate access keys
 
-[Read more](vars.md)
+## Variables
 
-All of the output values will be shown when you run the apply command except the ```secretKey``` value since that is a sensitive value so Terraform by default hides it. You can run ```terraform output --json``` to show all output values in JSON format which will also show the ```secretKey```.
+| Name                 | Description                                                                                                   | Type    | Default     | Required |
```
Comment (zubair-nabi) [inline aws/README.md:14]: Let's autogenerate these through this module: https://github.com/terraform-docs/terraform-docs and link the generate .md file.

File: aws/README.md [hunk 2]
```diff
@@ -38,22 +75,37 @@ With this method, events are sourced from AWS CloudTrail at an organizational le
...
 4. **SQS**: This queue receives events as described by the EventBridge rule. This is the queue our crawler will keep track of to ingest the latest events.
 
+Both approaches above make use of the default EventBridge for each AWS account, by adding a rule to the account you want to crawl. This rule forwards the events to the SQS queue provisioned by the `aws` terraform app. View the image below for a graphical overview (The image omits a couple of other resources provisioned by `aws` TF for the sake of simplicity in showing the 2 approaches):
+
+### Diagram
+
+![Event Crawl Diagram](https://github.com/Kaleidoscope-Inc/blueprint-configurator/assets/2979095/18ee9d76-c8c2-4871-984c-4e15133fae58)
+
+
+# Usage of this module
```
Comment (zubair-nabi) [inline aws/README.md:95]: Do you think it's clear to the reader that this module is going to produce an access key and secret key that needs to be used in the AWS blueprint configuration in Kaleidoscope?

File: azure/kscope_crawl/README.md [hunk 3]
```diff
@@ -1,57 +1,79 @@
...
 # Usage of this script
 
 This Terraform script defines infrastructure as code to deploy an Azure storage account and an Azure Active Directory (AD) application. It enables diagnostic logging and metrics for the storage account and provides outputs to retrieve important information about the created resources.
 
-## Overview
+## Prerequisites
 
-The script sets up an Azure storage account, container, and queue, and enables diagnostic logging and metrics for the storage account. It also creates an Azure AD application with a display name of `kaleidoscope-blueprint` and assigns an owner based on the `object_id` of the current client configuration. It specifies a required resource access to a specific scope. The script provides outputs to retrieve information about the created resources.
+- Terraform installed
+- Azure CLI configured and authenticated
```
Comment (zubair-nabi) [inline azure/kscope_crawl/README.md:14]: Similar to AWS, can we not take Azure credentials as vars?

File: azure/kscope_crawl/README.md [hunk 4]
```diff
@@ -1,57 +1,79 @@
...
+- Appropriate permissions in Azure Active Directory (Application Administrator or Global Administrator role)
 
-### Permissions required in Azure AD
-Please make sure that you have the 'Application Administrator' or 'Global Administrator' role in Azure Active Directory. Otherwise, the script will have insufficient privileges to update the Azure AD service principal with the specified object ID.
+## Variables
 
-Ensure that you have the appropriate permissions assigned and try running the Terraform operation again. If you are unsure about your permissions or need additional access, reach out to your Azure AD administrator or the appropriate security team within your organization to request the necessary privileges.
+| Name                   | Description                                                                      | Type   | Default Value | Required |
```
Comment (zubair-nabi) [inline azure/kscope_crawl/README.md:19]: Should we auto generate this from the vars file?

**Review by zubair-nabi — COMMENTED** [review]
Good work! Left some comments.

File: aws/README.md [hunk 5]
```diff
@@ -1,26 +1,63 @@
...
 The purpose of this Terraform module is to create resources in the AWS account that needs to be crawled by Kaleidoscope's AWS Blueprint.
 
 This README describes the usage of this module as well the resources that are created and their need.
 
-# Usage of this module
+## Prerequisites
 
-The ```main.tf``` file shows how to import this module. We use Terraform's concept of local modules for now.
+- Terraform installed (version >=1.1.9)
+- AWS CLI configured with appropriate access keys
```
[the remark this answers is not in the document]
Comment (waleedakramkhan) [reply aws/README.md:10]: oh i did not know that. fixed.

File: azure/kscope_crawl/README.md (hunk 3 above)
Comment (waleedakramkhan) [reply azure/kscope_crawl/README.md:14]: let me see

File: aws/README.md (hunk 2 above)
Comment (waleedakramkhan) [reply aws/README.md:95]: this was dangling. removed. last point of `Usage` clears that out.

File: azure/kscope_crawl/README.md (hunk 3 above)
Comment (waleedakramkhan) [reply azure/kscope_crawl/README.md:14]: done

File: azure/kscope_crawl/README.md (hunk 4 above)
Comment (waleedakramkhan) [reply azure/kscope_crawl/README.md:19]: done

**Review by zubair-nabi — APPROVED** [review]
LGTM! 

Great work!

**Review by hasnatbashir — APPROVED** [review]

**Review by muhammad-tahir-nawaz — APPROVED** [review]
LGTM!

**Pull request merged**